tests/reftests/clean.test

```diff
@@ -109,3 +110,175 @@ Remaining files:
   - ${BASEDIR}/OPAM/clean/bin/first
 
 Remove them? [y/n] y
+### : Clean of sources directory :
+### <waste/file>
+yes, i'm a file
+### <pin:waste/waste.opam>
+opam-version: "2.0"
+### tar czf waste.tgz waste/file
+### <pkg:waste.1>
+opam-version: "2.0"
+### <mkurl.sh>
+n=waste
+v=1
+arch=waste.tgz
+file="REPO/packages/$n/$n.$v/opam"
+echo "url {" >> $file
+echo "src: \"$arch\"" >> $file
+MD5=$(openssl md5 $arch | cut -d' ' -f2)
+echo "checksum: \"md5=$MD5\"" >> $file
+echo "}" >> $file
+basedir=`echo $BASEDIR | sed "s/\\\\\\\\/\\\\\\\\\\\\\\\\/g"`
+echo "dev-repo: \"file://${basedir}/$n\"" >> $file
+### sh mkurl.sh
+### <pkg:waste-local.1>
+opam-version: "2.0"
+### <mkurl.sh>
+n=waste-local
+v=1
+file="REPO/packages/$n/$n.$v/opam"
+basedir=`echo $BASEDIR | sed "s/\\\\\\\\/\\\\\\\\\\\\\\\\/g"`
+echo "url { local: \"file://${basedir}/waste\" }" >> $file
+### sh mkurl.sh
+### opam update
+
+<><> Updating package repositories ><><><><><><><><><><><><><><><><><><><><><><>
+[default] synchronised from file://${BASEDIR}/REPO
+Now run 'opam upgrade' to apply any package updates.
+### opam install waste
+The following actions will be performed:
+=== install 1 package
+  - install waste 1
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> retrieved waste.1  (file://${BASEDIR}/waste.tgz)
+-> installed waste.1
+Done.
+### ls OPAM/clean/.opam-switch/sources/
+waste.1
```

Comment:
This one's part of the original bug, right? After opam install, there's no need for it to be there?